uroot/root.go

```diff
@@ -143,14 +143,15 @@ var (
 		Dir{Name: "/usr/lib", Mode: os.FileMode(0777)},
 		Dir{Name: "/go/pkg/linux_amd64", Mode: os.FileMode(0777)},
 		// chicken and egg: these need to be there before you start and hence
-		// built into the initial initramfs. 
+		// built into the initial initramfs.
 		//{Name: "/dev/null", Mode: uint32(syscall.S_IFCHR) | 0666, dev: 0x0103},
 		//{Name: "/dev/console", Mode: uint32(syscall.S_IFCHR) | 0666, dev: 0x0501},
 		Dev{Name: "/dev/tty", Mode: uint32(syscall.S_IFCHR) | 0666, Dev: 0x0500},
 		Dev{Name: "/dev/urandom", Mode: uint32(syscall.S_IFCHR) | 0444, Dev: 0x0109},
 		Mount{Source: "proc", Target: "/proc", FSType: "proc", Flags: syscall.MS_MGC_VAL, Opts: ""},
 		Mount{Source: "sys", Target: "/sys", FSType: "sysfs", Flags: syscall.MS_MGC_VAL, Opts: ""},
 		Mount{Source: "cgroup", Target: "/sys/fs/cgroup", FSType: "tmpfs", Flags: syscall.MS_MGC_VAL, Opts: ""},
+		Mount{Source: "none", Target: "/tmp", FSType: "tmpfs", Flags: syscall.MS_MGC_VAL, Opts: ""},
```

Comment:
You can leave source empty (just ""), and you need not pass syscall.MS_MGC_VAL (hasn't been required since Linux 2.something)